packages/slate-editor/src/modules/editor/lib/file-attachment/handleEditAttachment.ts

```diff
@@ -15,16 +15,18 @@ export async function handleEditAttachment(editor: Editor, element: Partial<Atta
         return;
     }
 
-    Transforms.setNodes<AttachmentNode>(editor, element, {
+    const newElement = { ...currentFileAttachment, ...element };
+
+    newElement.file.filename = newElement.file.filename.replaceAll('/', ':');
```

Comment:
1. Should we maybe use `_` instead of `:` ? It's a safer choice for a filename IMO.
2. I have a bad feeling about mutating `file` property object. This can lead to something we do not expect. 
  Should we maybe use a safer update strategy? 


```tsx
export async function handleEditAttachment(editor: Editor, update: Partial<AttachmentNode>) {
   // ...
   const element = mapFilename(
      { ...currentFileAttachment, ...update },
      (filename) => filename.replaceAll('/', '_'),
   );
   // ...
}

function mapFilename(element: AttachmentNode, map: (filename: string) => string): AttachmentNode {
   return {
      ...element,
     file: {
        ...element.file,
        filename: map(element.file.filename),
     }
   };
}
```